staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go

```diff
@@ -410,6 +413,16 @@ func startFakeKMSProvider(version, endpoint string) (*fakeKMSPlugin, error) {
 type fakeKMSPlugin struct {
 	apiVersion string
 	server     *grpc.Server
+	sockFile   string
+}
+
+func (s *fakeKMSPlugin) Stop() {
+	// Stop the server
+	s.server.Stop()
+	// If this isn't a Linux abstract namespace socket, or if we're on a non-linux platform, clean up the socket file
+	if !strings.HasPrefix(s.sockFile, "@") || runtime.GOOS != "linux" {
```

Comment:
@liggitt the file is marked `// +build !windows` ... do we leave it as-is?

Reply:
yes